Form/Handler/AuthorizeFormHandler.php

```diff
@@ -23,10 +24,10 @@ class AuthorizeFormHandler
     protected $request;
     protected $form;
 
-    public function __construct(FormInterface $form, Request $request)
+    public function __construct(FormInterface $form, RequestStack $requestStack)
     {
         $this->form = $form;
-        $this->request = $request;
+        $this->request = $requestStack->getMasterRequest();
```

Comment:
Please don't get the request in the constructor. This is broken as your service is not request-scoped and so might be instantiated before the request is available for instance.
You should store the RequestStack in a private property instead, and retrieve the current request only in ``process()``